react/props.md

```diff
@@ -151,15 +193,15 @@ const App = () => {
 export default App
 ```
 
-The variable `name` is assigned information from the value in state. This makes `name` available to `Greeter` as props. Since `Greeter` is a display component there are no updates that need to happen in order to handle our refactor in `App.js`.
+The variable `name` is now assigned information from state rather than hardcoded. Since the Greeter component is a display component, no updates are needed for our refactor in `App.js`.
 
 ### Mapping a Component Call
 
-Our greeter application is working just great. But what if we have multiple people to greet? To expand the functionality of the application we can set an array in state. Then rather than a single `Greeter` component invocation we want to have one component per person. To manage this dynamically we can map over the array in state and return a component invocation. Now, no matter how many people we need to greet our code can manage the job effectively.
+Our greeter application is working just great. But what if we have multiple people to greet? To expand the functionality of the application we can set an array in state. Instead of a single Greeter component invocation, we will have one component per person. To manage this dynamically, we will map over the array in state and return a component invocation. No matter how many people we need to greet now our code can manage the job effectively.
```

Comment:
```suggestion
Our greeter application is working just great. But what if we have multiple people to greet? To expand the functionality of the application we can set an array in state. Instead of a single Greeter component invocation, we will have one component per person. To manage this dynamically, we will map over the array in state and return a component invocation. No matter how many people we need to greet, now our code can manage the job effectively.
```